src/types/table.ts

```diff
@@ -76,3 +76,15 @@ export interface TablePaginationAttributes {
 }
 
 export type RowBulkAction = boolean | { enabled: boolean, disabledTooltip?: string }
+
+export interface DataSelectState {
```

Comment:
nit: should this be prefixed with `Table` like the other interfaces?

Reply:
Updated (`DataSelectState` -> `TableViewSelectState`)